packages/firebase_messaging/firebase_messaging/example/lib/message.dart

```diff
@@ -78,47 +77,47 @@ class MessageView extends StatelessWidget {
                   ),
                   row(
                     'Channel ID',
-                    notification.android.channelId,
+                    notification.android!.channelId,
```

Comment:
`notification` is nullable but I don't see a null check for it, how is that possible?